tests/functional/workflows/nodejs_npm/test_utils.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+import sys
+
+import os
+
+import shutil
+
+import tempfile
+
+from aws_lambda_builders.workflows.nodejs_npm import utils
+
+
+@pytest.fixture
+def osutils():
+    return utils.OSUtils()
+
+
+class TestOSUtils(object):
```

Comment:
This test class is using pytest but others are using unittest. Can we keep everything on one framework (preferable unittest)

Reply:
actually, this test class was copied from `python_pip/utils.py`, which was copied from Chalice's code. So this has to be cleaned up anyway. I would not spend more time diverging the two if possible.

Reply:
resolved with https://github.com/gojko/aws-lambda-builders/commit/ed23979bc52ed87b6b2e1e3700d7051dcbd4b0f1